src/data.js

```diff
@@ -5334,6 +5334,23 @@ module.exports = [
       'Swift',
     ],
   },
+  {
+    name: 'Sumanth',
+    description:'Student. Learning web development',
+    url: 'https://mynameissumanth.netlify.app/uses.html',
+    country: 'in',
```

Comment:
Hey @enigmacoder-bot will you please use the flag emoji for your country?

My only other request would be adding a bit more to your uses page on your site. Right now it's pretty barebones.